tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserSBytePerfTests.cs

```diff
@@ -58,6 +82,30 @@ private static void PrimitiveParserByteSpanToSByte_BytesConsumed(string text)
         }
 
         [Benchmark(InnerIterationCount = InnerCount)]
+        private static void PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength()
```

Comment:
Can you add back the baseline tests for these too and show the results (comparison)?